schemas/mozilla_nimbus_schemas/experiments/experiments.py

```diff
@@ -109,6 +109,16 @@ class ExperimentMultiFeatureMobileBranch(BaseExperimentMultiFeatureBranch):
     """
 
 
+class ExperimentLocalizations(RootModel[dict[str, dict[str, str]]]):
```

Comment:
Oh wild so you can declare the actual type using the generic type syntax right at the class definition level.  That's crazy I didn't know that.

Reply:
new in pydantic v2! I think you could also do `ExperimentLocalizations = RootModel[...]` but then you dont have the docstring attached